run/2bot-simulator.cfg

```diff
@@ -1,28 +0,0 @@
-<simulation>
```

Comment:
LMFAO this shouldn't even exist. I actually didn't even know it did.

Reply:
Agree. I wonder if there's a way to change grSim settings programmatically...probably not but it would be nice.